src/compute_surf.cpp

```diff
@@ -104,15 +103,19 @@ ComputeSurf::~ComputeSurf()
   delete [] which;
   memory->destroy(array_surf_tally);
   memory->destroy(tally2surf);
-  delete hash;
-  memory->destroy(vector_surf);
```

Comment:
I had to remove this line or it crashes with a segfault.

Reply:
yes, whack vector_surf